data/test/index.js

```diff
@@ -12,3 +20,40 @@ describe( 'store', () => {
 		expect( store2.getState() ).toEqual( 'ribs' );
 	} );
 } );
+
+describe( 'select', () => {
+	it( 'registers multiple selectors to the public API', () => {
+		const store = registerReducer( 'reducer1', () => 'state1' );
+		const selector1 = jest.fn( () => 'result1' );
```

Comment:
What do you think about using `registerSelectors` in this test and `registerSelector` in the one testing `query`?